x/tx/CHANGELOG.md

```diff
@@ -31,6 +31,90 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
```

Comment:
As said in the other PR, changelog was misplaced here, this should all be reverted